openedx/core/djangoapps/content/course_overviews/tests/factories.py

```diff
@@ -13,9 +13,12 @@ class Meta(object):
         django_get_or_create = ('id', )
 
     version = CourseOverview.VERSION
-    pre_requisite_courses = []
     org = 'edX'
 
+    @property
+    def pre_requisite_courses(self):
+        return []
```

Comment:
Was this because there were some tests mutating the array attribute in a way that leaked between tests?  Will those tests start failing themselves because they can no longer mutate this directly?

Reply:
This was a change that slipped in when I moved these over from another PR. I'll remove it as it doesn't address the issue I was poking at.